Dockerfile

```diff
@@ -1,9 +1,9 @@
-FROM mcr.microsoft.com/dotnet/sdk:5.0 AS build
+FROM mcr.microsoft.com/dotnet/sdk:8.0 AS build
 WORKDIR /app
 COPY . ./
 RUN dotnet publish -c Release -o out
 
-FROM mcr.microsoft.com/dotnet/aspnet:5.0
```

Comment:
Pretty sure this can be changed to `mcr.microsoft.com/dotnet/runtime:8.0`. Did a quick scan, and it doesn't look like anything references aspnet.